src/gdc_ng_models/models/download_reports.py

```diff
@@ -104,45 +112,105 @@ class DataDownloadReport(Base):
         db.DateTime(timezone=True), nullable=False, server_default=db.text("now()")
     )
 
-    def add_access_type(self, access_type, size):
+    def add_size_access_type(self, access_type, size):
         """
         Args:
             access_type (str): open/closed
             size (double): size in GB
         """
         if not self.access_type_report:
             self.access_type_report = {}
-        self.access_type_report[access_type] = size
+        if access_type not in self.access_type_report:
+            self.access_type_report[access_type] = DataDownloadReport._create_default()
+        self.access_type_report[access_type][SIZE_FIELD] = size
 
-    def add_experimental_strategy(self, strategy, size):
+    def add_size_experimental_strategy(self, strategy, size):
         """
         Args:
             strategy (str): strategy name
             size (double): size in GB
         """
         if not self.experimental_strategy_report:
             self.experimental_strategy_report = {}
-        self.experimental_strategy_report[strategy] = size
+        if strategy not in self.experimental_strategy_report:
+            self.experimental_strategy_report[strategy] = (
+                DataDownloadReport._create_default()
+            )
+        self.experimental_strategy_report[strategy][SIZE_FIELD] = size
 
-    def add_project_id(self, project, size):
+    def add_size_project_id(self, project, size):
```

Comment:
I'm assuming this is adding the downloaded size in GB for the project?

Reply:
Yes. Your assumption is correct.
The second field added (with their respective helper methods) is download counts.